File: 백준/Bronze/3040. 백설 공주와 일곱 난쟁이/백설 공주와 일곱 난쟁이.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        ArrayList<Integer> num = new ArrayList<>();
        Scanner sc = new Scanner(System.in);
        for (int i = 0; i < 9; i++) {
            num.add(sc.nextInt());
        }
        
        int MUST = 100;
        int summ = 0;
        for (int n : num) {
            summ += n;
        }
        Loop1:
        for (int i = 0; i < 9; i++) {
            for (int j = i+1; j < 9; j++) {
                if (summ - num.get(i) - num.get(j) == MUST) {
                    int numi = num.get(i);
                    int numj = num.get(j);
                    num.remove(Integer.valueOf(numi));
                    num.remove(Integer.valueOf(numj));
                    break Loop1;
                }
            }
        }
        for (int x : num) {
            System.out.println(x);
        }
    }
}

File: 백준/Bronze/3040. 백설 공주와 일곱 난쟁이/백설 공주와 일곱 난쟁이.py
num = []
for _ in range(9):
    num.append(int(input()))

MUST = 100
summ = sum(num)
flag = True
for i in range(9):
    for j in range(i+1, 9):
        if summ - num[i] - num[j] == MUST:
            numi, numj = num[i], num[j]
            num.remove(numi)
            num.remove(numj)
            flag = False
            break
    if flag == False:
        break
            
for x in range(7):
    print(num[x])